formats/role.jsonnet

```diff
@@ -49,9 +49,26 @@
         supports_historical_accounts: {
           type: "boolean",
         },
-        seniority :{
+        seniority: {
           type: "integer",
         },
+	      whip_organisation: {
+	       type: "object",
```

Comment:
Still looks like some tabs have crept in 😅 I don't usually have this problem myself because VS Code fixes space formatting on save (I think due to [this extension](https://marketplace.visualstudio.com/items?itemName=chenxsan.vscode-standardjs)). There may be an equivalent for Vim...?

Reply:
Oh my god just saw this review, will fix now 👍